packages/tests/stuff/training_stuff.sh

```diff
@@ -27,6 +27,7 @@ TEST_DATA_TI_ID=training-data-helper
 EXAMPLES_VERSION=$(jq '.examples_version' -r "${CLUSTER_PROFILE}")
 CLOUD_PROVIDER="$(jq '.cloud.type' -r "${CLUSTER_PROFILE}")"
 CLUSTER_NAME="$(jq '.cluster_name' -r "${CLUSTER_PROFILE}")"
+BUCKET_NAME="$(jq '.data_bucket' -r "${CLUSTER_PROFILE}")"
```

Comment:
Why did you decide to delete '-data-store' part of the bucket name?
Or you don't?

Reply:
I've decided to use bucket name from profile and do not use some hardcoded naming convention.